README.md

```diff
@@ -6,27 +6,12 @@
 
 # Prefect Recipes 🧑‍🍳 🥐
 
-This repository contains common and extensible Prefect patterns to drive efficient workflows &mdash; we like to call these patterns our **recipes**!
+This repository contains common and extensible Prefect patterns to drive efficient workflows &mdash; we like to call these patterns [our **recipes**](#table-of-contents-📖)!
```

Comment:
I think you'll need to remove the book emoji for the link to work, but keep the dash after contents

i.e. `[our **recipes**](#table-of-contents-)!`

Reply:
Ah, annoying that VSC preview does not recognize links the same way GitHub does. Fixing...